cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -170,11 +166,23 @@ def _check_file_size(file_path, expected_size):
     temp_appliance_preconfig.db.restore_database(db_path, is_major=is_major)
 
 
+@pytest.fixture
+def create_po_user_and_group(request, appliance):
+    """This fixture creates custom user with tenant attached"""
+    group = _groups(
+        request,
+        appliance,
+        appliance.rest_api.collections.roles.get(name="EvmRole-super_administrator"),
+        description="Preserve Owner Report Group",
+    )
+    _users(request, appliance, group=group.description, userid="pouser")
```

Comment:
**Question** : Do we want to cleanup these user/groups afterwards?

Reply:
Nope, that will be done by the `cfme.rest.gen_data._creating_skeleton` function which is used by `_users`.